README.md

```diff
@@ -20,6 +20,36 @@ var Writable = zstreams.Writable,
 The base zstreams function like, and inherit from, the native Node.JS streams.  They mostly behave like their
 parents, with
 
+### Simplified Constructors
```

Comment:
This whole section should probably be moved down.  A readme should be organized according to how pertinent the information is to typical use cases.  Someone is a lot more likely to need to convert a native stream than to use a simplified constructor format (for example).